cmd/timestamp-server/app/serve.go

```diff
@@ -122,4 +122,9 @@ var serveCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(serveCmd)
 	rootCmd.AddCommand(version.Version())
+
+	serveCmd.Flags().Int64("max-request-body-size", 1048576, "Maximum allowed size for request bodies in bytes (1MB by default)")
```

Comment:
Can we initialize this with the other server flags under https://github.com/sigstore/timestamp-authority/blob/5c83c88b6013fb881a54bdd1637144dac734ab99/cmd/timestamp-server/app/root.go#L56?